core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -62,6 +62,10 @@ class KryoSerializerSuite extends FunSuite with SharedSparkContext {
     val thrown3 = intercept[IllegalArgumentException](new KryoSerializer(conf4).newInstance())
     assert(thrown3.getMessage.contains(kryoBufferProperty))
     assert(!thrown3.getMessage.contains(kryoBufferMaxProperty))
+    val conf5 = conf.clone()
+    conf5.set(kryoBufferProperty, "8m")
```

Comment:
Currently there is no test with legitimate kryoBufferProperty value expressed in 'm'

This addition fills the gap.